google-cloud-error_reporting/lib/google/cloud/error_reporting/error_event.rb

```diff
@@ -0,0 +1,312 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
You want 2 new lines between the copyright comments and the first bit of code. Otherwise the copyright notice will be included as the documentation for the Google constant. And every file will add a duplicate copyright to the constant's documentation.